LEAF_Request_Portal/ajaxIndex.php

```diff
@@ -216,6 +216,7 @@ function customTemplate($tpl)
 
         break;
     case 'cancel':
+        /* This endpoint has been deprecated as of 8/31/2023 */

```

Comment:
Please also open an issue to track future cleanup of deprecated endpoints. Example: https://github.com/department-of-veterans-affairs/LEAF/issues/2062